src/rest/main/main.go

```diff
@@ -95,17 +104,17 @@ func main() {
 		}
 	}()
 
-	if caFile == "" && server.ClientAuth.Enabled("cert") {
+	if caFile == "" && clientAuth.Enabled("cert") {
 		glog.Fatal("Must specify -cacert with -client_auth cert")
 	}
 
 	swagger.Load()
 
 	server.GenerateJwtSecretKey()
-	server.JwtRefreshInt = time.Duration(jwtRefInt * uint64(time.Second))
-	server.JwtValidInt = time.Duration(jwtValInt * uint64(time.Second))
+	server.JwtRefreshInt = time.Duration(30 * time.Second)
```

Comment:
We should document the fact that token validity/refresh times are hardcoded in this release and make sure everyone's aware of this as well. Can you update the HLD?

Reply:
Okay I will address in the HLD